src/main/java/uk/gov/ons/census/casesvc/model/entity/Case.java

```diff
@@ -126,6 +126,9 @@
   @Column(columnDefinition = "BOOLEAN DEFAULT false")
   private boolean handDelivery;
 
+  @Column(columnDefinition = "BOOLEAN DEFAULT false")
+  private boolean skellington;
```

Comment:
while I do find this amusing, I think it will make it very hard for DAP to use our data, and for the testers and others to find the column they're looking for in our database, let alone anybody who's searching our code for the _skeleton_ marker.